pureskillgg_dsdk/tome/constants.py

```diff
@@ -2,20 +2,19 @@
 import structlog
 
 
-def get_tome_manifest_key_fs(path):
-    return os.path.join(path, "tome")
-
-
-def get_tome_path_fs(root_path, prefix, tome_name):
-    tome_parent_path = (
-        root_path if prefix is None else os.path.join(root_path, *prefix.split("/"))
+def get_tome_manifest_key_fs(path, prefix, ds_type, tome_name, is_copied_header=False):
+    return os.path.join(
+        path,
+        prefix if prefix is not None else "",
```

Comment:
Doesn't this result in a double `/`, like `"/".join("a", "", "b") == "a//b" NOT "a/b` right? Unless os.path.join works differently. You can do a conditional push instead maybe?, e.g.

```python
parts = [path]
if prefix is not None:
  parts.append(prefix)

return os.path.join(*parts)
```

Same for the header line.

Reply:
Right, it does path normalization: 
![Untitled](https://user-images.githubusercontent.com/7268131/173168900-890c488b-9988-4935-824d-ca854a090a0f.png)

Reply:
Oh this particular function is unused now. For joining path, prefix, and key, I have copied the same func we used for the ds_io stuff.

Reply:
unused so i delete.